mmv1/products/iap/api.yaml

```
@@ -237,3 +237,125 @@ objects:
         description: |
           Human-friendly name given to the OAuth client.
         required: true
+  - !ruby/object:Api::Resource
+    name: 'Settings'
+    base_url: '{{name}}:iapSettings'
+    create_url: 'projects/{{project}}/iap_web/compute/services/{{backend_service_id}}:iapSettings'
+    self_link: 'projects/{{project}}/iap_web/compute/services/{{backend_service_id}}:iapSettings'
+    create_verb: :PATCH
+    update_verb: :PATCH
+    description: |
+      The IAP configurable settings.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      api: 'https://cloud.google.com/iap/docs/reference/rest/v1/IapSettings'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'backend_service_id'
+        description: |
+          Identifier of the backend service to which this IAP settings should be attached to.
+        input: true
+        output: false
+        required: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          Required. The resource name of the IAP protected resource.
```

```suggestion
         The resource name of the IAP protected resource.
```

If you feel it should be kept anyways, we'll want to remove `Required` since the field is output-only!